### ISSUE TITLE
tslint was deprecated in favour of eslint

### ISSUE THREAD

**Issue by silkentrance** [testdeck/testdeck#308]
We must use eslint instead of tslint.

**Comment by silkentrance** [comment]
@PanayotCankov what kind of code style do you prefer?

```
error Strings must use singlequote
error Unexpected empty arrow function 
warning 'done' is defined but never used
error Type string trivially inferred from a string literal, remove type annotation
```

and so on.

- 2 vs 4 characters as tab?

**Comment by silkentrance** [comment]
@PanayotCankov see also https://github.com/testdeck/testdeck/actions/runs/3120165126/jobs/5060555532

**Comment by silkentrance** [comment]
This will be fixed in #315
